backend/audit/templates/audit/my_submissions.html

```diff
@@ -49,33 +46,28 @@ <h2>Start a new submission</h2>
                 <input id="check-start-new-submission" class="usa-checkbox__input" type="checkbox" />
                 <label class="usa-checkbox__label" for="check-start-new-submission">
                     I agree to the above
-                    <button
-                        id="terms-conditions-trigger"
-                        class="usa-button--unstyled margin-top-0"
-                        aria-controls="modal-terms-conditions"
-                        data-open-modal
-                    >terms and
+                    <button id="terms-conditions-trigger" class="usa-button--unstyled margin-top-0"
```

Comment:
There are a lot of linebreak changes in this PR, and this one in particular looks wrong, but I think most of them aren't related to the actual changes at hand. Can you back those out and just keep the relevant changes?

If we need to add a linting/formatting standard for the files, we can do that separately; there is a Python package, `djlint`, for this, although I haven't investigated it closely.

Reply:
@tadhg-ohiggins My editor (VSCode) seems to like formatting the code this way. black is formatting python files correctly but we don't have something for html templates yet. I'm wondering if I should backout changes to this file completely and have @jasnakai re-implement the column ordering as there is another PR that was merged yesterday which has changes to this html template.

Reply:
I think backing out these changes makes sense; reordering the columns can be a very small future PR.